core_app/mls_integration.py

```
@@ -0,0 +1,314 @@
+                _mls_data[mls_num] = row.to_dict()
+        
+        _mls_last_loaded = datetime.now().isoformat()
+        
+        return {
+            'success': True,
+            'count': len(_mls_data),
+            'message': f'Successfully loaded {len(_mls_data)} MLS listings',
+            'last_updated': _mls_last_loaded
+        }
+        
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)